src/Serializers/ClaimsSerializer.php

```diff
@@ -20,7 +20,7 @@ class ClaimsSerializer implements DispatchableSerializer {
 	/**
 	 * @var Serializer
 	 */
-	private $claimSerializer;
+	private $statementSerializer;
```

Comment:
Considering that this is package private, and that it'll be removed once the deprecated `newClaimSerializer` gets removed, you are likely putting in effort that will not help us with anything